core/src/main/java/hudson/model/Queue.java

```diff
@@ -1814,18 +1814,22 @@ public Api getApi() {
         /**
          * Returns true if the execution should be blocked
          * for temporary reasons.
-         *
-         * <p>
-         * Short-hand for {@code getCauseOfBlockageForItem()!=null}.
+         * @deprecated Use {@link #getCauseOfBlockage} != null
```

Comment:
As of 2.85 was already not being called by core, so why keep it?

Reply:
Should it be also restricted?